spec/fixtures/api/gdk-backend-check/groot.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
```

Comment:
are all of these fields necessary? can we trim out some of them?

Reply:
desktop entry must have a `Type` and a `Name` key. Our use case needs `Exec` and `MimeType`.
We can remove `Terminal`, as it doesn't affect our use case, not mandatory as well.